WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/network/NetworkUtils.java

```diff
@@ -245,6 +245,13 @@ public static String buildExistingProjectURL(String id) {
 	public static String buildIntervalsPostURL(String userid, String projectid) {
 		return buildExistingUserURL(userid) + "/" + projectid + "/intervals";
 	}
+	
+	/**
+	 * @return the URL to post new events to the server to for this user.
+	 */
+	public static String buildEventsPostURL(String userid, String projectid) {
+		return buildExistingUserURL(userid) + "/" + projectid + "/events";
```

Comment:
This the only real difference I can make out between events and intervals, right?